docs/content/release_notes/csi_rn_limitations.md

```diff
@@ -29,4 +29,11 @@ The following limitations apply when expanding volumes with the IBM block storag
 
 -   When using the CSI driver with IBM Spectrum Virtualize Family and IBM DS8000 Family products, during size expansion of a PersistentVolumeClaim (PVC), the size remains until all snapshots of the specific PVC are deleted.
 -   When expanding a PVC while not in use by a pod, the volume size immediately increases on the storage side. PVC size only increases, however, after a pod begins to use the PVC.
--   When expanding a filesystem PVC for a volume that was previously formatted but is now no longer being used by a pod, any copy or replication operations performed on the PVC (such as snapshots or cloning, and so on) results in a copy with the newer, larger, size on the storage. However, its filesystem has the original, smaller, size.
\ No newline at end of file
+-   When expanding a filesystem PVC for a volume that was previously formatted but is now no longer being used by a pod, any copy or replication operations performed on the PVC (such as snapshots or cloning) results in a copy with the newer, larger, size on the storage. However, its filesystem has the original, smaller, size.
+
+## Volume replication limitations
+
+When a role switch is conducted, this is not reflected within the other orchestration platform replication objects.
+
+**Important:** When using volume replication, be sure to delete the PersistentVolume applying the `Retain` reclaim policy and using the latest import procedure (version 1.7.0 or later) (see [Importing an existing volume](../configuration/csi_ug_config_advanced_importvol.md) in the User Guide).
```

Comment:
**only for volumes created with a driver version lower than 1.7.0**, the steps are:
1. change reclaim policy of the relevant PVs to `Retain`
2. delete said PVs
3. import them using a procedure of 1.7.0 or later

IMO this steps order is not clear from current articulation

Reply:
DONE